libs/designer/src/lib/ui/templates/filters/templateFilters.tsx

```diff
@@ -64,13 +78,17 @@ export const TemplateFilters = ({ connectors, detailFilters }: TemplateFiltersPr
         />
       </div>
       <div className="msla-templates-filters-dropdowns">
-        {connectors && (
+        {allUniqueConnectors && allUniqueConnectors.length > 0 && (
```

Comment:
Not at the moment, do we want to show the loading instead of the dropdown - would that UI make sense? Or should we just show the empty dropdown until the connectors load; this might be a design discussion